moderator/settings.py

```diff
@@ -146,66 +112,77 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
     'django.contrib.auth.hashers.UnsaltedMD5PasswordHasher',
 )
 
+# Django browserid authentication backend
+AUTHENTICATION_BACKENDS = (
+    'moderator.moderate.backend.ModeratorBrowserIDBackend',
+    'django.contrib.auth.backends.ModelBackend',
+)
+
 HMAC_KEYS = {
     '2015-08-17': 'pancakes',
 }
 
 PASSWORD_HASHERS = get_password_hashers(BASE_PASSWORD_HASHERS, HMAC_KEYS)
 
+SESSION_SERIALIZER = 'django.contrib.sessions.serializers.JSONSerializer'
 
-# Django browserid authentication backend
-AUTHENTICATION_BACKENDS = (
-    'moderator.moderate.backend.ModeratorBrowserIDBackend',
-    'django.contrib.auth.backends.ModelBackend',
-)
 
-SITE_URL = 'https://moderator.mozilla.org'
-ALLOWED_HOSTS = [SITE_URL]
-BROWSERID_AUDIENCES = [SITE_URL]
+# Absolute filesystem path to the directory that will hold user-uploaded files.
+# Example: "/var/www/example.com/media/"
+MEDIA_ROOT = path('media')
+# URL that handles the media served from MEDIA_ROOT. Make sure to use a
+# trailing slash.
+# Examples: "http://example.com/media/", "http://media.example.com/"
+MEDIA_URL = '/media/'
+# Absolute path to the directory static files should be collected to.
+# Don't put anything in this directory yourself; store your static files
+# in apps' "static/" subdirectories and in STATICFILES_DIRS.
+# Example: "/var/www/example.com/static/"
+STATIC_ROOT = path('static')
+STATIC_URL = '/static/'
+
+# Internationalization
+TIME_ZONE = config('TIME_ZONE', default='UTC')
+USE_I18N = config('USE_I18N', default=False, cast=bool)
+USE_L10N = config('USE_L10N', default=False, cast=bool)
+USE_TZ = config('USE_TZ', default=True, cast=bool)
+
+SECRET_KEY = config('SECRET_KEY')
+DEBUG = config('DEBUG', default=False, cast=bool)
+SITE_URL = config('SITE_URL', default='http://127.0.0.1:8000')
```

Comment:
I would prefer to avoid having localhost as default, instead we can define it in .env